src/python/scripts/twitter_scrooge_pluck_cmd.py

```diff
@@ -0,0 +1,48 @@
+import os
+import re
+import sys
+
+def parse_gen_file_map(filename):
+  r = re.compile("(\S+) -> (\S+)")
+  with open(filename) as file:
+    m = {}
+    for line in file:
+      g = r.match(line)
+      k = g.group(1)
+      v = m.get(k, set())
+      v.add(g.group(2))
+      m[k] = v
+  return m
+
+def main(jar_cmd, manifest, output_jar, cmd_file, srcjar_polluted, gen_file_map, include_base, pluck_base, files):
+
+  gen_map = parse_gen_file_map(gen_file_map)
+  #TODO we should just walk srcjar_polluted, and only include mentioned sources
```

Comment:
I see we still have srcjar_polluted, but I'm not sure why we need it. Given that creating those files is not free, I would say we should try to minimize the jars we create.